src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -744,19 +742,23 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& stream, c
         _logger.debug("Successfully read %zu bytes into blob.", graphSize);
 
         auto meta = compiler->parse(blob, localConfig);
+        // graphHandle is reused by backend if we use driver compiler
```

Comment:
// If graphHandle is not a nullptr it means there is still an instance of the blob maintained inside the driver and we can release the copy of the blob here to reduce memory consumption.